plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueConverter.java

```diff
@@ -139,6 +139,11 @@ public static DatabaseInput toGlueDatabaseInput(Database database)
     }
 
     public static Table fromGlueTable(software.amazon.awssdk.services.glue.model.Table glueTable, String databaseName)
+    {
+        return fromGlueTable(glueTable, databaseName, false);
+    }
+
+    public static Table fromGlueTable(software.amazon.awssdk.services.glue.model.Table glueTable, String databaseName, boolean ignoreDeltaLakeColumns)
```

Comment:
`ignoreDeltaLakeColumns` -> `ignoreColumns`

We may be dealing similarly with an Iceberg or Hudi table

Reply:
I can't write a test for other connectors.

Reply:
Let's drop `boolean ignoreDeltaLakeColumns` parameter.
I think we can agree that if `isDeltaLakeTable(tableParameters)` should be sufficient to allow us to ignore the columns.

FWIW I'm ok with approving the PR with this requested change.